packages/testsuite/cypress/support/containers-utils.ts

```diff
@@ -3,11 +3,27 @@ Cypress.Commands.add("startWildflyContainer", () => {
     "start:wildfly:container",
     {
       name: Cypress.spec.name.replace(/\.cy\.ts/g, "").replace(/-/g, "_"),
+      configuration: "standalone-insecure.xml",
+      useNetworkHostMode: false,
     },
     { timeout: 240_000 }
   );
 });
 
+Cypress.Commands.add("startWildflyContainerSecured", (configuration) => {
+  return cy.task("start:wildfly:container", {
+    name: Cypress.spec.name.replace(/\.cy\.ts/g, "").replace(/-/g, "_"),
+    configuration: configuration,
+    useNetworkHostMode: true,
+  });
+});
+
+Cypress.Commands.add("startSsoContainer", (name) => {
+  return cy.task("start:sso:container", {
+    name: name,
```

Comment:
Why do we need the name parameter? Can we use a constant instead?

Reply:
generate name like for WF